drafts/2018-02-13-this-week-in-rust.md

```diff
@@ -14,6 +14,8 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 
 # Updates from Rust Community
 
+The community team tries to get in touch with meetup organisers better. Please fill out their [call for contact info](https://docs.google.com/forms/d/e/1FAIpQLSf52YXGhqBaHtCXtVna4iHYMK7IQaTqUW6V-ztsZC8C2TBInQ/viewform) if you are running or used to run a meetup.
```

Comment:
"The community team is trying to improve outreach to meetup organizers." (more idiomatic english)

Reply:
Fixed, thanks!